github.go

```diff
@@ -224,20 +281,23 @@ func (m *GithubClient) GetPullRequest(prNumber, commitRef string) (*PullRequest,
 		"repositoryOwner": githubv4.String(m.Owner),
 		"repositoryName":  githubv4.String(m.Repository),
 		"prNumber":        githubv4.Int(pr),
-		"commitsLast":     githubv4.Int(100),
+		"commitsLast":     githubv4.Int(1),
 	}
 
 	// TODO: Pagination - in case someone pushes > 100 commits before the build has time to start :p
 	if err := m.V4.Query(context.TODO(), &query, vars); err != nil {
 		return nil, err
 	}
+
 	for _, c := range query.Repository.PullRequest.Commits.Edges {
+		prObj := PullRequest{
```

Comment:
Please revert the changes in this `for` loop this so there is no diff, since there are no real code changes anymore after you've moved your list function.

Reply:
done